Common/Securities/CashBook.cs

```diff
@@ -31,10 +31,10 @@ namespace QuantConnect.Securities
     public class CashBook : IDictionary<string, Cash>
     {
         /// <summary>
-        /// Gets the base currency used
+        /// Base currency used
         /// </summary>
-        public const string AccountCurrency = "USD";
-
+        public string AccountCurrency { get; set; } = "USD";
```

Comment:
I think if we're going to make this instance based, the it should be a required constructor argument to force callers to set it properly. This also sheds light on the initialization issue, that is to say,  I think we need to know the account currency _before_ initializing the algorithm